resources/geometry.rb

```diff
@@ -517,48 +507,43 @@ def self.process_occupants(model, runner, num_occ, occ_gain, sens_frac, lat_frac
     people_sch = nil
     activity_sch = nil
 
-    # Get spaces
-    ffa_spaces = self.get_conditioned_spaces(model.getSpaces)
-
-    ffa_spaces.each do |space|
-      space_obj_name = "#{Constants.ObjectNameOccupants}|#{space.name.to_s}"
-
-      space_num_occ = num_occ * UnitConversions.convert(space.floorArea, "m^2", "ft^2") / cfa
+    space_obj_name = "#{Constants.ObjectNameOccupants}|#{space.name.to_s}"
```

Comment:
I would get rid of this line now and just use `Constants.ObjectNameOccupants` now. Since there's only one object created, no need to name it based on the space its associated with.